cluster/saltbase/salt/kubelet/default

```diff
@@ -84,4 +84,4 @@
   {% set pod_cidr = "--pod-cidr=" + grains['cbr-cidr'] %}
 {% endif %} 
 
-DAEMON_ARGS="{{daemon_args}} {{api_servers_with_port}} {{debugging_handlers}} {{hostname_override}} {{cloud_provider}} {{config}} --allow_privileged={{pillar['allow_privileged']}} {{pillar['log_level']}} {{cluster_dns}} {{cluster_domain}} {{docker_root}} {{kubelet_root}} {{configure_cbr0}} {{cgroup_root}} {{system_container}} {{pod_cidr}}"
+DAEMON_ARGS="{{daemon_args}} {{api_servers_with_port}} {{debugging_handlers}} {{hostname_override}} {{cloud_provider}} {{config}} --allow-privileged=true {{pillar['log_level']}} {{cluster_dns}} {{cluster_domain}} {{docker_root}} {{kubelet_root}} {{configure_cbr0}} {{cgroup_root}} {{system_container}} {{pod_cidr}}"
```

Comment:
@thockin, I know there are other places, but here are the reasons I didn't modify other places:

1) cluster/saltbase/pillar/privilege.sls: I cannot change this one because kube-apiserver use this value too. 
2) hardcoded value in cluster/saltbase/salt/kubelet/default, so that GCE, GKE, AWS (not coreos one), and other first line of cloud provider has the flag set for kubelet default for 1.0.
3) coreos is nice-to-have for 1.0. I decided to leave the as it. Like you mentioned, those places have to be hard-coded in cloud_config file.